openfisca_web_api_preview/openAPI.yml

```diff
@@ -0,0 +1,253 @@
+    get:
+      tags:
+      - "Parameters"
+      summary: "List all available parameters"
+      operationId: "getParameters"
+      produces:
+      - "application/json"
+      responses:
+        200:
+          description: "The list of parameters is sent back in the response body"
+          headers :
+            Country-Package :
+              description : "The name of the country package served by the API"
+              type: "string"
+            Country-Package-Version:
+              description: "The version of the country package served by the API"
+              type: "string"
+          schema:
+            $ref: '#/definitions/Parameters'
+  /parameter/{parameterID}:
+    get:
+      tags:
+      - "Parameters"
+      summary: "Gets information about a specific parameter"
+      operationId: "getParameter"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "parameterID"
+        in: "path"
+        description: "ID of parameter (e.g. impot_revenu.bareme)"
+        required: true
+        type: "string"
+      responses:
+        200:
+          description: "The requested parameter's information is sent back in the response body"
+          headers :
+            Country-Package :
+              description : "The name of the country package served by the API"
+              type: "string"
+            Country-Package-Version:
+              description: "The version of the country package served by the API"
+              type: "string"
+          schema:
+            $ref: "#/definitions/Parameter"
+        404:
+          description: "The requested parameter does not exist"
+          headers :
+            Country-Package :
+              description : "The name of the country package served by the API"
+              type: "string"
+            Country-Package-Version:
+              description: "The version of the country package served by the API"
+              type: "string"
+  /variables:
+    get:
+      tags:
+      - "Variables"
+      summary: "List all available variables"
+      operationId: "getVariables"
+      produces:
+      - "application/json"
+      responses:
+        200:
+          description: "The list of variables is sent back in the response body"
+          headers :
+            Country-Package :
+              description : "The name of the country package served by the API"
+              type: "string"
+            Country-Package-Version:
+              description: "The version of the country package served by the API"
+              type: "string"
+          schema:
+            $ref: "#/definitions/Variables"
+  /variable/{variableID}:
+    get:
+      tags:
+      - "Variables"
+      summary: "Gets information about a specific variable"
+      operationId: "getVariable"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "variableID"
+        in: "path"
+        description: "ID of a variable (e.g. aide_embauche_pme)"
```

Comment:
This example is misleading, as this country package might not provide it, and the user who would try to use this example would face a 404 and not understand what went wrong. Explain that these IDs can be obtained by enumerating the `/variables` endpoint.